build.gradle

```diff
@@ -22,6 +22,7 @@ compileJava {
     }
 }
 
+//Run tests and code coverage tools
 test {
     useJUnitPlatform()
```

Comment:
I guess you removed my changes for "compile" to "implementation" for the dependencies? That's fine, but "compile" is deprecated now I believe, so I was just updating how it should look.